### PR TITLE
finish scoreboard page

### PR DESCRIPTION
**Pull request by kornkritpawit**

### CODE REVIEW

**Comment by Klan300** [comment]
every thing ok

File: Cardgame/admin.py [hunk 1]
```diff
@@ -14,4 +14,5 @@ class DeckAdmin(admin.ModelAdmin):
 
     list_display = ('deck_name','number_of_card')
 
-admin.site.register(Deck, DeckAdmin)
\ No newline at end of file
+admin.site.register(Deck, DeckAdmin)
+admin.site.register(Playerscore)
```
Comment (Klan300) [inline Cardgame/admin.py]: delete this code because playerscore cannot add or create

**Review by Klan300 — CHANGES_REQUESTED** [review]
fix some code and wait our team review before merge.

File: Cardgame/admin.py (hunk 1 above)
Comment (Klan300) [reply Cardgame/admin.py]: กรอันนี้กู merge เอง

**Comment by Parkkung** [comment]
can't use play again button,

**Comment by kornkritpawit** [comment]
> can't use play again button,

fixed

File: Cardgame/admin.py (hunk 1 above)
Comment (kornkritpawit) [reply Cardgame/admin.py]: ok krub

**Review by Klan300 — APPROVED** [review]

**Pull request merged**